apps-rendering/src/components/Layout/GalleryLayout.tsx

```diff
@@ -1,15 +1,21 @@
 // ----- Imports ----- //
 import { css } from '@emotion/react';
-import { neutral } from '@guardian/source-foundations';
+import { from, neutral } from '@guardian/source-foundations';
```

Comment:
strange the linter hasn't complained, but my IDE indicates `from` isn't used in this file

Reply:
TS has complained, the validation check is failing.